erpnext/shopping_cart/doctype/gateway_selector_item/gateway_selector_item.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
"Gateway Selector Item" is not a good name of the doctype, maybe use "Shopping Cart Payment Gateway"